packages/trpc/server/routers/viewer/availability/schedule/update.handler.ts

```diff
@@ -78,13 +78,16 @@ export const updateHandler = async ({ input, ctx }: UpdateOptions) => {
     };
   }
 
+  const timeBlocks = input.timeBlocks?.filter((timeBlock) => timeBlock !== "");
+
```

Comment:
Hey, can we test this in the client-side input field as well? I don't see a check for empty blocks in there. Currently, I can save with an empty input block. I know the blocks aren't saved, but it would make more sense to show an error alert instead of saving and just showing a toast message.

Reply:
Showing an error alert if an empty time block text field is submitted.